openslides_backend/presenter/base.py

```diff
@@ -0,0 +1,13 @@
+presenters = {}
+
+
+def register_presenter(name):
+    def wrapper(c):
```

Comment:
Just a stylish remark: I prefer self explaning variable names not just single characters like `c`. See the same function in action. Please also add typings.

Reply:
See https://github.com/OpenSlides/openslides-backend/pull/40/commits/213db5f4841c9b12a29f0ed2c22d44e2bb5b3091